0035-ilp-over-http/0035-ilp-over-http.md

```diff
@@ -52,7 +59,72 @@ Content-Type: application/octet-stream
 < Body: Binary OER-Encoded ILP Fulfill or Reject Packet >
 ```
 
-All ILP Packets MUST be returned with the HTTP status code `200: OK`.
+If the request included a `Request-Id` header, the recipient handling the ILP Prepare SHOULD choose to handle the packet asynchronously and return the corresponding ILP Fulfill/Reject in a separate outgoing HTTP request.
+
+In the asynchronous mode, if the request is semantically valid, the recipient MUST respond immediately that the ILP Prepare is accepted for processing, even if the packet will ultimately be rejected:
+
+```http
+HTTP/x.x 202 Accepted
+```
+
+#### Retry Behavior
+
+Two peers MAY retry ILP Prepare packets, but MUST both agree to support retries out-of-band to prevent duplicate side effects.
+
+If the recipient of the ILP Prepare responds with an `5xx` or `409 Conflict` status, or no HTTP response is received within a given timeout, the sender of the ILP Prepare SHOULD retry sending the packet with the same idempotency key.
+
+The sender MUST conclude retrying after receiving a response with a `2xx` status, `4xx` error, or receiving the corresponding ILP Fulfill/Reject packet.
+
+The sender SHOULD ensure there are multiple attempts to deliver the packet to the peer before the ILP Prepare expires.
+
+### Async ILP Fulfill/Reject Reply
+
+#### Request
+
+```http
+POST /ilp HTTP/x.x
+Host: example.com
+Content-Type: application/octet-stream
+Authorization: Bearer zxcljvoizuu09wqqpowipoalksdflksjdgxclvkjl0s909asdf
+Request-Id: 42ee09c8-a6de-4ae3-8a47-4732b0cbb07b
+Idempotency-Key: 6ff99499-008e-4499-8644-048450627496
+
+< Body: Binary OER-Encoded ILP Fulfill or Reject Packet >
+```
+
+- **Path** &mdash; A connector MAY specify any HTTP path for their peer to send ILP packets to.
+- **Host Header** &mdash; The standard [HTTP Host Header](https://tools.ietf.org/html/rfc2616#section-14.23) indicating the domain of the HTTP server the Request is sent to.
+- **Content-Type Header** &mdash; MUST be set to `application/octet-stream`.
+- **Request Id Header** &mdash; Request ID from the corresponding ILP Prepare, which is a UUIDv4.
+- **Idempotency Key Header** &mdash; _Optional_. UUIDv4 to uniquely identify this packet.
```

Comment:
could clarify with "Uniquely identify this Fulfill/Reject packet"

Reply:
https://github.com/interledger/rfcs/compare/09916ea8c1232f846610940066298f052f140c8f...8b43dafce2c84d073bc70afacf396f700d4ff8f5#diff-ccfda9b7df50a9cfd8618bb9c08e360aR99